src/events.c

```diff
@@ -412,6 +412,12 @@ static bool events_handler_key_down(SDL_Event *event, void *arg) {
 		}
 	}
 
+	if (!repeat) {
+		if(scan == config_get_int(CONFIG_KEY_TOGGLE_AUDIO)) {
+			config_set_int(CONFIG_MUTE_AUDIO, !config_get_int(CONFIG_MUTE_AUDIO));
```

Comment:
This probably should be moved into [`events_handler_hotkeys`](https://github.com/taisei-project/taisei/blob/8b7a259759e7cdb7f336af3bf894fa2bd6f6c30e/src/events.c#L440).